cn/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,118 @@
+---
+assignees:
+- bgrant0607
```

Comment:
If any (future) PR touches this file, @k8s-merge-robot will randomly pick one of the assignees specified here, and assign that person as reviewer